cxx-checks/src/test/resources/checks/UsingNamespaceInHeader.h

```diff
@@ -30,3 +30,13 @@ namespace Foo
         privateFunction( );
     }
 }
+
+void fooFunc()
```

Comment:
clang-tidy causes warning here and below too. Both have rule IDs `google-build-using-namespace` aka  "do not use namespace using-directives; use using-declarations instead". So that's some separate warning, which is not specific to the headers. Therefore I think your fix works as expected. Thank you.